editor/animation_track_editor.cpp

```diff
@@ -5160,9 +5180,8 @@ void AnimationTrackEditor::_update_step(double p_new_step) {
 	double step_value = p_new_step;
 	if (timeline->is_using_fps()) {
 		if (step_value != 0.0) {
-			// step_value must also be less than or equal to 1000 to ensure that no error accumulates due to interactions with retrieving values from inner range.
+			// Step_value must also be less than or equal to 1000 to ensure that no error accumulates due to interactions with retrieving values from inner range.
```

Comment:
`step_value` is the name of the variable, can keep it lowercase for variable name consistency.